README.md

```diff
@@ -32,9 +33,44 @@ For alternative ways, stop by our Community Hub [stackrox.io](https://www.stackr
     + [How to Deploy](#how-to-deploy)
 * [Generating portable installers](#generating-portable-installers)
 
-## Deployment
+## Quick Installation via Helm
 
-To quickly deploy the latest development version of StackRox to your kubernetes
+StackRox offers quick installation via Helm Charts, follow the [Helm Installation Guide](https://helm.sh/docs/intro/install/) for system specific instructions and requirements.
+
+First, add the [stackrox/helm-charts/opensource](https://github.com/stackrox/helm-charts/tree/main/opensource) repository to Helm.
+```sh
+helm repo add stackrox-oss https://raw.githubusercontent.com/stackrox/helm-charts/main/opensource/
+```
+To see all available Helm charts in the repo run (you may add the option `--devel` to show non-release builds as well)
+```sh
+helm search repo stackrox-oss
+```
+From here you can install stackrox-central-services to get Central and Scanner components deployed on your cluster. Note that you need only one deployed instance of stackrox-central-services even if you plan to secure multiple clusters.
+```sh
+helm install -n stackrox-oss --create-namespace stackrox-central-services stackrox-oss/stackrox-central-services --devel
+```
+To create a secured cluster you first need to create an init bundle containing all of the secrets.
+```sh
+kubectl -n stackrox exec deploy/central -- roxctl --insecure-skip-tls-verify \
+    --password "$(kubectl -n stackrox get secret central-htpasswd -o go-template='{{index .data "password" | base64decode}}')" \
+    central init-bundles generate stackrox-init-bundle --output - > stackrox-init-bundle.yaml
+```
+Then install Secured Cluster Services in the same cluster using this command:
```

Comment:
Maybe let's use "stackrox-secured-cluster-services" instead of "Secured Cluster Services" to be consistent with "stackrox-central-services" you set before.
```suggestion
Then install stackrox-secured-cluster-services in the same cluster using this command:
```

Reply:
Done